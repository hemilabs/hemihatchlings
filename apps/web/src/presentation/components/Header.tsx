
function Header(): JSX.Element {
  return (
    <header className="flex flex-row w-full justify-center items-center h-24 gap-x-8
      sm:h-28 lg:h-32">
      <div>
        <h1 className="font-poller text-white text-lg
          sm:text-xl lg:text-2xl">HH</h1>
      </div>
      <div className="grow border h-0 border-b-0 max-h-0 max-w-5xl w-auto border-white"></div>
      <div>
        <ul className="font-monaco text-white text-sm text-right
          sm:text-lg lg:text-xl">
          <a href="https://github.com/hemilabs"
            target='_blank'
            className="hover:text-gray-400"><li>GITHUB</li></a>
          <a href="https://discord.gg/RyhaPp7NvQ"
            target='_blank'
            className="hover:text-gray-400">
            <li>COMMUNITY</li>
          </a>
          <a href="https://twitter.com/hemi_xyz/"
            target='_blank'
            className="hover:text-gray-400">
            <li>TWITTER</li>
          </a>
        </ul>
      </div>
    </header>
  )
}

export default Header
