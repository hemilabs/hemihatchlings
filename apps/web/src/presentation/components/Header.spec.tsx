import { describe, it, expect} from 'vitest'
import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

describe('src/presentation/Header', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Header />)

    expect(asFragment()).toMatchSnapshot()
  })
})
