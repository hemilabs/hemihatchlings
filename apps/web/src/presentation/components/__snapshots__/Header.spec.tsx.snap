// Vitest Snapshot v1

exports[`src/presentation/Header > renders correctly 1`] = `
<DocumentFragment>
  <header
    class="flex flex-row w-full justify-center items-center h-24 gap-x-8 sm:h-28 lg:h-32"
  >
    <div>
      <h1
        class="font-poller text-white text-lg sm:text-xl lg:text-2xl"
      >
        HH
      </h1>
    </div>
    <div
      class="grow border h-0 border-b-0 max-h-0 max-w-5xl w-auto border-white"
    />
    <div>
      <ul
        class="font-monaco text-white text-sm text-right sm:text-lg lg:text-xl"
      >
        <a
          class="hover:text-gray-400"
          href="https://github.com/hemilabs"
          target="_blank"
        >
          <li>
            GITHUB
          </li>
        </a>
        <a
          class="hover:text-gray-400"
          href="https://discord.gg/RyhaPp7NvQ"
          target="_blank"
        >
          <li>
            COMMUNITY
          </li>
        </a>
        <a
          class="hover:text-gray-400"
          href="https://twitter.com/hemi_xyz/"
          target="_blank"
        >
          <li>
            TWITTER
          </li>
        </a>
      </ul>
    </div>
  </header>
</DocumentFragment>
`;
